fastlane/testfile.swift

```diff
@@ -0,0 +1 @@
+struct TestFile {}
```

Comment:
What's this for? Could we have a comment at the start of this file to explain?

Reply:
removed here 043ccb3d0683ec005a700b09ae205021866998de